strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/LocalStorageProxyRepositoryExpiredArtifactsCleaner.java

```diff
@@ -84,7 +84,8 @@ private void filterAccessibleProxiedArtifacts(final List<ArtifactEntry> artifact
         for (final Iterator<ArtifactEntry> it = artifactEntries.iterator(); it.hasNext(); )
         {
             final ArtifactEntry artifactEntry = it.next();
-            final Storage storage = configurationManager.getConfiguration().getStorage(artifactEntry.getStorageId());
+            final Storage storage = configurationManager.getConfiguration().getStorage(
+                    artifactEntry.getStorageId());
```

Comment:
Cosmetic: Could be on the previous line.

Reply:
done